src/controllers/usersController.js

```diff
@@ -185,7 +186,7 @@ export const signin = async (req, res) => {
           });
         })();
       } else {
-        return errorRes(res, 400, res.__('Incorrect password'));
+        return errorRes(res, 409, res.__('Incorrect Email or password'));
```

Comment:
why did you change the status code to 409? it should be 400

Reply:
yeah sure thanks Ray, i can respond with a bad request instead of conflicts